packages/engine/src/simulation/package/context/mod.rs

```diff
@@ -103,7 +107,15 @@ impl ContextColumn {
     }
 }
 
+/// Provides the functionalities of writing a column into the context batch. Implementing this 
+/// trait allows the creation of trait-objects so that the root context package can call the 
+/// writing functionality in whatever order it needs, and therefore the other context packages do 
+/// not need to be aware of one another.
```

Comment:
```suggestion
/// Provides the functionalities of writing a column into the context batch.
///
/// Implementing this trait allows the creation of trait-objects so that the root context package can call the writing functionality in whatever order it needs, and therefore the other context packages do not need to be aware of one another.
```